rfc/rfc019-employee-identity-means.md

```diff
@@ -0,0 +1,187 @@
+
+This RFC is an addition to the means listed in [RFC002](./rfc002-authentication-token.md#7-supported-means)
+
+### Status of document
+
+This document is currently in draft.
+
+### Copyright Notice
+
+![](../.gitbook/assets/license.png)
+
+This document is released under the [Attribution-ShareAlike 4.0 International \(CC BY-SA 4.0\) license](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## 1.  Introduction
+
+Introducing new authentication means in an organizational setting can have quite an impact, even more when mainstream identity providers have not yet adjusted to new standards.
+New feature or applications require a higher assurance level for authentication than the current situation.
+This introduces an additional authentication step in the process of users. This often happens at a place in the workflow where they don't expect it.
+This is a situation that remain until the main login is adapted to the newer/higher security standard.
+This RFC bridges the gap so the software can already be changed to accept newer protocols and the user can be made aware of upcoming changes without going through an authentication step using additional means.
+
+## 2. Terminology
+
+* **Client application**: The application that requires access.
+* **Node**: A piece of software implementing the Nuts RFCs.
+* **Resource server**: The application \(a protected resource\) that requires authorized access to its API’s.
+* **VerifiableCredential**: Verifiable Credential according to the [Verifiable Credential Data Model](https://www.w3.org/TR/vc-data-model/).
+* **VerifiablePresentation**: Verifiable Presentation according to the [Verifiable Credential Data Model](https://www.w3.org/TR/vc-data-model/).
+
+## 3. Nuts Employee Identity
+
+The authentication flow is as follows:
+
+- a user requests access to a resource that is hosted at a *resource server* and requires additional authentication.
+- the *client application* starts a session on the *node* with the current user data.
+- the user is directed to a web page where it accepts the given conditions by pressing a button.
+- this button initiates a POST request to the *node for the given session.
+- upon receiving the request, the node will issue a `NutsEmployeeCredential` from and to the organization DID of the current user context.
+- the credential is used to create a VerifiablePresentation.
+- the VerifiablePresentation is used in the access token request as defined in [RFC003](./rfc003-oauth2-authorization.md#422-payload).
+
+### 3.1 User data
+
+In order for the resource server to comply to local legislation the following data MUST be added to the credential:
+
+- **Initials**: The initials of the user.
+- **Familyname**: The family name of the user.
+- **Identifier**: A unique identifier for the user within the context of the organization. This may be an email address or an employee number. 
+
+The user data MAY also contain a `RoleName` that contains the user role within the organization.
```

Comment:
```suggestion
The user data MAY also contain a `roleName` that contains the user role within the organization.
```

If using backticks here (so naming the exact field), do the same for the 3 above